components/invitation_modal/invitation_modal_guest_step/index.ts

```diff
@@ -32,4 +32,8 @@ function mapDispatchToProps(dispatch: Dispatch) {
     };
 }
 
-export default connect(mapStateToProps, mapDispatchToProps)(InvitationModalGuestsStep);
+const connector = connect(mapStateToProps, mapDispatchToProps);
+
+export type PropsFromRedux = ConnectedProps<typeof connector>;
```

Comment:
Interesting. I didn't know about that helper before. I'm curious if it even types `props.actions.getSubscriptionStats` properly when it's a thunk action

Reply:
yes it does can you check below
<img width="591" alt="image" src="https://user-images.githubusercontent.com/17708702/129688555-517eee99-2f45-4ce7-893f-cc42ffa0a180.png">

Reply:
That's not quite correct then actually. `connect` should get rid of the `ActionFunc` return value